src/dodal/devices/xbpm_feedback.py

```diff
@@ -11,18 +11,14 @@ class XBPMFeedback(Device):
     STABILITY_TIME = 3
 
     pos_ok: EpicsSignalRO = Component(EpicsSignalRO, "-EA-FDBK-01:XBPM2POSITION_OK")
+    pos_stable: EpicsSignalRO = Component(EpicsSignalRO, "-EA-FDBK-01:XBPM2_STABLE")
     pause_feedback: EpicsSignal = Component(EpicsSignal, "-EA-FDBK-01:FB_PAUSE")
     x: EpicsSignalRO = Component(EpicsSignalRO, "-EA-XBPM-02:PosX:MeanValue_RBV")
     y: EpicsSignalRO = Component(EpicsSignalRO, "-EA-XBPM-02:PosY:MeanValue_RBV")
 
     def trigger(self) -> StatusBase:
-        status = StableSubscriptionStatus(
-            self.pos_ok,
+        return SubscriptionStatus(
+            self.pos_stable,
             lambda value, *args, **kwargs: value == 1,
```

Comment:
[nit, ignorable]

> [Signature should be `f(*, old_value, value, **kwargs)`](https://blueskyproject.io/ophyd/user_v1/generated/ophyd.status.SubscriptionStatus.html#ophyd.status.SubscriptionStatus)

I know this is technically correct but maybe it would be clearer to use the exact signature advised in the ophyd docs?